api/discovery/apis.json

```diff
@@ -281,6 +281,10 @@
         {
           "groupVersion": "flowcontrol.apiserver.k8s.io/v1beta3",
           "version": "v1beta3"
+        },
+        {
+          "groupVersion": "flowcontrol.apiserver.k8s.io/v1beta2",
+          "version": "v1beta2"
```

Comment:
Is this change relevant to this PR?

Reply:
I think this is just due to a stale openapi spec

Let me re-run `hack/update-openapi-spec.sh`

Reply:
Removed these changes.